sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -2134,6 +2134,11 @@ def _from_generated(cls, data):
             symbol=data.currency_symbol,
         )
 
+    def __str__(self):
+        if self.symbol is not None:
+            return f"{self.symbol}{self.amount}"
```

Comment:
is this always like "$1.00" or can it be, for example, "USD1.00"?

Reply:
It should only be the symbol as in "$"